tests/e2e/e2e_test.go

```diff
@@ -37,6 +37,11 @@ import (
 
 // TODO: Find more scalable way to do this
 func (s *IntegrationTestSuite) TestAllE2E() {
+	// If we skip the upgrade, we need to give our nodes time for their RPCs to come online, otherwise we get errors.
+	if s.skipUpgrade {
+		time.Sleep(3 * time.Second)
```

Comment:
I'm sure there is a better way we can fix this, but just setting this in place for now until after CL when I go back into this